tests/utils/test_validation.py

```diff
@@ -270,7 +270,7 @@ def test_validate_batch_log_data(monkeypatch):
     ]
     tags_with_bad_val = [
         RunTag("good-tag-key", "hi"),
-        RunTag("another-good-key", "but-bad-val" * 1000),
+        RunTag("another-good-key", "but-bad-val" * 6000),
```

Comment:
```suggestion
        RunTag("another-good-key", "a" (MAX_TAG_VAL_LENGTH + 1)),
```

It's unclear whether `"but-bad-val" * 6000)` really exceeds the limit.